src/NZGBplugin/LINZ/Widgets/DictionaryList.py

```diff
@@ -9,353 +9,366 @@
 #
```

Comment:
Is this a style-only change ? All this style changes make reviews very very hard to make

Reply:
No, missing  PyQt5.QtWidgets import added

I would like to get this all merged into master and fix styling as the next step so that we do not need to deal with this again